internal/plugins/ansible/v1/init.go

```diff
@@ -176,11 +176,11 @@ func addInitCustomizations(projectName string) error {
 	// todo: remove it when we solve the issue operator-framework/operator-sdk#3573
 	const resourcesLimitsFragment = `  resources:
```

Comment:
@camilamacedo86 fyi this will not be added into the deployment, it will be removed here: https://github.com/operator-framework/operator-sdk/pull/5330/files#diff-528ef321c70eee98f0bd420be05ef20020fcfd28f6b546cc516ed65381cce5d5L185

Reply:
@fabianvf we are not changing the Ansible scaffolds at all here.
Note that the ansible plugin uses the same customize config and then we remove the resources limits/resources for Ansible. 
Since the default values changed in KB we just need to change that here to still do the same (removing), You can check that it was not added at all by looking at the testdata/ansible samples which must be updated in the PR otherwise we are unable to pass in the sanity test.

Reply:
Regarding the times then still the same, see; 11 and 21 min:

<img width="654" alt="Screenshot 2021-11-17 at 00 17 26" src="https://user-images.githubusercontent.com/7708031/142086556-48a6e419-dcbf-4a70-aa81-e27c95822635.png">

<img width="657" alt="Screenshot 2021-11-17 at 00 17 16" src="https://user-images.githubusercontent.com/7708031/142086559-0b4e2089-5bf0-4225-b13f-e7fc3290f614.png">